### PR TITLE
Updated router links

### PR DESCRIPTION
**Pull request by djek195**
1. created constant values for Route
2. added links for footer and logos
3. small update footer styles

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -28,8 +29,8 @@ function App() {
           <BrowserRouter>
             <Header />
             <Routes>
-              <Route path="/" element={<CatalogPage />} />
-              <Route path="/category" element={<CategoryPage />} />
+              <Route path={RoutesLinks.HOMEPAGE} element={<CatalogPage />} />
+              <Route path={RoutesLinks.CATEGORYPAGE} element={<CategoryPage />} />
```
Comment (dimalian) [inline src/App.js]:
```suggestion
              <Route path={RoutesLinks.CATEGORY_PAGE} element={<CategoryPage />} />
```

File: src/constant/constant.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const RoutesLinks = {
+  HOMEPAGE: '/store-app',
+  CATEGORYPAGE: '/category',
```
Comment (dimalian) [inline src/constant/constant.js]:
```suggestion
  CATEGORY_PAGE: '/category',
```

**Review by dimalian — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (djek195) [reply src/App.js]: done

File: src/constant/constant.js (hunk 2 above)
Comment (djek195) [reply src/constant/constant.js]: done

**Pull request merged**